poly-synth/src/poly-synth-demo.js

```diff
@@ -362,6 +488,22 @@ class PolySynthDemo {
           default: this.masterGain_.gain.value
         });
   }
+  
+  getDrumSampleUrls_() {
+    let drumSampleUrls =
```

Comment:
Again, this creates an array every time the method called. You should have this in a global data storage for this class.